R/create_template.R

```diff
@@ -104,7 +106,81 @@
 #'         General sections are called as child documents in this skeleton and
 #'         each of the child documents should be edited separately.
 #' @export
-#' @examples create_template()
+#'
+#' @examples
+#' \dontrun{
+#' create_template(
+#'   new_section = "a_new_section",
+#'   section_location = "before-introduction",
+#'   )
+#'
+#'
+#' create_template(
+#'   new_template = TRUE,
+#'   format = "pdf",
+#'   office = "NWFSC",
+#'   species = "Dover sole",
+#'   spp_latin = "Microstomus pacificus",
+#'   year = 2010,
+#'   author = c("John Snow", "Danny Phantom", "Patrick Star"),
+#'   include_affiliation = TRUE,
+#'   resdir = "C:/Users/Documents/Example_Files",
+#'   model_results = "Report.sso",
+#'   model = "SS3",
+#'   new_section = "an_additional_section",
+#'   section_location = "after-introduction",
+#'   )
+#'
+#' asar::create_template(
+#'   new_template = TRUE,
+#'   format = "pdf",
+#'   office = "PIFSC",
+#'   species = "Striped marlin",
+#'   spp_latin = "Kajikia audax",
+#'   year = 2018,
+#'   author = "Alba Tross",
+#'   model = "BAM",
+#'   new_section = c("a_new_section", "another_new_section"),
+#'   section_location = c("before-introduction", "after-introduction"),
+#'   custom = TRUE,
+#'   custom_sections = c("executive_summary", "introduction")
+#' )
+#'
+#' create_template(
+#'   new_template = TRUE,
+#'   format = "pdf",
+#'   office = "NWFSC",
+#'   region = "my_region",
+#'   complex = FALSE,
+#'   species = "Bluefish",
+#'   spp_latin = "Pomatomus saltatrix",
+#'   year = 2010,
+#'   author = c("John Snow", "Danny Phantom", "Patrick Star"),
+#'   add_author = "Sun E Day",
+#'   include_affiliation = TRUE,
+#'   simple_affiliation = TRUE,
+#'   alt_title = FALSE,
+#'   title = "Management Track Assessments Spring 2024",
+#'   parameters = TRUE,
+#'   param_names = c("region", "year"),
+#'   param_values = c("my_region", "2024"),
+#'   convert_output = FALSE,
+#'   fleet_names = c("fleet1", "fleet2", "fleet3"),
+#'   resdir = "C:/Users/Documents/Example_Files",
+#'   model_results = "Report.sso",
+#'   model = "SS3",
+#'   new_section = "an_additional_section",
+#'   section_location = "before-discussion",
+#'   type = "SAR",
+#'   prev_year = 2021,
+#'   custom = TRUE,
+#'   custom_sections = c("executive_summary", "introduction", "discussion"),
+#'   include_figures = TRUE,
+#'   include_tables = TRUE,
+#'   add_image = TRUE,
+#'   spp_image = "dir/containing/spp_image"
+#' )
+#' }
```

Comment:
I'm not sure if this is the standard format, but I have seem examples that also end with the comment of don't run so people don't forget lol